### PR TITLE
Libretto theme

### PR DESCRIPTION
**Pull request by abrahamvarricatt**
Adding a new theme for Nikola. Hope this gets accepted. :)

### CODE REVIEW

File: v7/libretto/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+Feel free to report any issues, make suggestions or even better - contribute!
+
+
+# Un-supported features
+
+Here is a list of Nikola features (that I'm aware of) which are currently not
+supported with this theme,
+
+* 3rd-party comment system  (is it even possible to do this without JS ?)
+* Tags
```
Comment (Kwpolska) [inline v7/libretto/README.md]: Why not?

File: v7/libretto/assets/xml/rss.xsl [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Kwpolska) [inline v7/libretto/assets/xml/rss.xsl]: No need to duplicate this file.

**Comment by Kwpolska** [comment]
The theme looks pretty good, but tags are pretty important. Could you please work on making them work? Leaving the built-in template from base would work better than an empty page.

File: v7/libretto/README.md (hunk 1 above)
Comment (abrahamvarricatt) [inline v7/libretto/README.md]: The original theme didn't have any UI for it. I didn't want to 'improvise' too much. Already did that for the archives part.

File: v7/libretto/assets/xml/rss.xsl (hunk 2 above)
Comment (abrahamvarricatt) [inline v7/libretto/assets/xml/rss.xsl]: I'm not inheriting from base for this theme. It's a 'from scratch' implementation. On my browser at least, things work better with this file included.

**Comment by abrahamvarricatt** [comment]
@Kwpolska   Hmmm... alright. Give me a few days to figure out how best to represent tags and I'll add those template files too. 

@ralsina  Thanks for merging this in so quickly! :)

**Pull request merged**